serverless/event_sources/knative-event-sources.adoc

```diff
@@ -5,14 +5,14 @@ include::modules/common-attributes.adoc[]
 :context: knative-event-sources
 toc::[]
 
-An _event source_ is a resource object that acts as a link between an event producer and a _sink_. A sink can be a Knative service, channel or broker that receives events from an event source.
+An _event source_ is a resource object that acts as a link between an event producer and an event  _sink_, or consumer. A sink can be a Knative Service, Channel, or Broker that receives events from an event source.
 
 Currently, {ServerlessProductName} supports the following event source types:
 
 ApiServerSource:: Connects a sink to the Kubernetes API server.
 PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
 
-SinkBinding is also supported, which allows you to connect core Kubernetes resources such as `Deployment`, `Job`, or `StatefulSet` with a sink.
+SinkBinding is also supported, which allows you to connect core Kubernetes resources such as Deployment, Job, or StatefulSet with a sink.
```

Comment:
But what is a SinkBinding, exactly? Without any additional context, I'm not sure what this means.

Reply:
This must be related to *Using SinkBinding*.

Reply:
Added an xref, thanks